sdk/communication/Azure.Communication.CallingServer/tests/Infrastructure/CallingServerTestEnvironment.cs

```diff
@@ -15,13 +15,18 @@ public class CallingServerTestEnvironment : CommunicationTestEnvironment
         /// <summary>
         /// The phone number associated with the source.
         /// </summary>
-        public string SourcePhoneNumber => GetRecordedVariable(AzurePhoneNumber, options => options.IsSecret());
+        public string SourcePhoneNumber => GetRecordedVariable(AlternateCallerId, options => options.IsSecret());
 
         /// <summary>
         /// The phone number associated with the source.
```

Comment:
Should this be The phone number associated with the target.?

Reply:
Good catch!         
Change it to:
/// The phone number required to make a pstn call.